import_activity_events.py

```diff
@@ -40,6 +40,12 @@
 EVENTS_FILE_FIXED_URL = "s3://" + EVENTS_BUCKET + "/" + EVENTS_PREFIX_FIXED + "events-{day}.fixed.csv"
 EVENTS_BEGIN = datetime.strptime("2016-09-02", "%Y-%m-%d")
 
+SAMPLE_RATES = (
+    {"percent":10, "months":24, "suffix":"_sampled_10"},
+    {"percent":50, "months":6, "suffix":"_sampled_50"},
+    {"percent":100, "months":3, "suffix":""}
+)
```

Comment:
This might deserve a couple of lines commend explaining what it's for and how it maps to the tables in the db.